sdk/resourcemanager/azure-resourcemanager-containerinstance/src/main/java/com/azure/resourcemanager/containerinstance/models/ContainerGroup.java

```diff
@@ -1179,7 +1182,18 @@ interface WithDnsPrefix {
              * @param dnsPrefix the DNS prefix to be used to create the FQDN for the container group
              * @return the next stage of the definition
              */
-            WithCreate withDnsPrefix(String dnsPrefix);
+            WithDnsNameLabelReusePolicy withDnsPrefix(String dnsPrefix);
+        }
+
+        /** The stage of the container group definition allowing to specify the DNS name label reuse policy. */
+        interface WithDnsNameLabelReusePolicy {
```

Comment:
`WithAutoGeneratedDomainNameLabelScope`

Reply:
Fixed in the [commit#6e748a3](https://github.com/Azure/azure-sdk-for-java/pull/42795/commits/6e748a33351eeda68205b4a03d69db268dd3d0ed)